borg/testsuite/helpers.py

```diff
@@ -259,6 +259,40 @@ def testInvalidUnicode(self):
         assert e.match(str(b"ba\x80/foo", 'latin1'))
 
 
+@pytest.mark.parametrize("lines, expected", [
+    ([], None),
+    (["# Comment only"], None),
+    (["*"], []),
+    (["# Comment",
+      "*/something00.txt",
+      "  whitespace\t",
+      "/whitespace/at/end of filename \t ",
+      # Whitespace before comment
+      " #/ws*",
```

Comment:
just noticed that this was existing behaviour. so let it as it is, we need to do this separately.